### PR TITLE
enable gkehub feature membership test for the new scifi controller

### PR DESCRIPTION
**Pull request by ziyue-101**
### Change description

enable gkehub feature membership test for the new scifi controller

### Tests you have done

- [ ] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: pkg/test/constants/constants.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ var TestNameRegexesToSkip = []string{
 	// TODO(b/220357089): re-enable eventfunction test in long-running CRUD test suite.
 	".*(eventfunction).*",
 	// TODO(b/215781076): re-enable gke hub tests in long-running CRUD test suite.
-	".*(gkehubfeaturemembership|mcifeature).*",
+	".*(mcifeature).*",
```
Comment (jingyih) [inline pkg/test/constants/constants.go:33]: Do we know how long the `gkehubfeaturemembership` tests take? The comment says it needs to be added to long-running.

Comment (ziyue-101) [reply pkg/test/constants/constants.go:33]: It will take ~20 min as the test creates a gke cluster

**Pull request closed without merging**